tests/test/generic/linker_namespace1.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python2.7
```

Comment:
Is this file used somewhere?

Reply:
I removed it, it was a debug file, and not supposed to get integrated.